lib/app_web/router.ex

```diff
@@ -13,9 +13,11 @@ defmodule AppWeb.Router do
 
   pipeline :api do
     plug CORSPlug,
-      origin: System.get_env("ALLOW_API_ORIGINS")
-              |> String.replace("'", "")
-              |> String.split(",")
+      origin:
+        System.get_env("ALLOW_API_ORIGINS")
```

Comment:
I don't think this should be an Environment Variable if we are just going to set it to **`*`** on Heroku ... 💭

Reply:
My thought for having it as an environment variable is that we could decide and remove quickly domain names that send request to the server. This could be a first way to filter unwanted request. However we can use also API keys.

Reply:
@SimonLab the only way to avoid bad actors at an API level is having API keys. 
Having a "whitelist" or "blacklist" of domains is worthless because requests can be _spoofed_. 
IP address whitelisting is viable because it works at the TCP level. 
Let's not worry about this now. We can always refactor it later. 👍